Plugins/ExternalTexture/Source/ExternalTexture_Shared.h

```diff
@@ -95,7 +95,7 @@ namespace Babylon::Plugins
                         return;
                     }
 
-                    runtime.Dispatch([deferred = std::move(deferred), handle, &context, impl = std::move(impl)](Napi::Env env) {
+                    runtime.Dispatch([deferred = std::move(deferred), handle, &context, impl = std::move(impl)](Napi::Env env) mutable {
```

Comment:
How is it that mutable is needed now but wasn't before? It doesn't seem like any actual code changed.